### ISSUE TITLE
Notifications not display "&" correctly

### ISSUE THREAD

**Issue by Kurt-Git** [strawberrymusicplayer/strawberry#584]
**Describe the bug**
The Pop Up notification doesn't translate the "&" sign correctly. It displays it as "amp;". 

**To Reproduce**
For instance have an artist of "Stevie Ray Vaughan & Double Trouble". This will be displayed as "Stevie Ray Vaughan amp; Double Trouble"

**Expected behavior**
I would expect "Stevie Ray Vaughan & Double Trouble"

**System Information:**
 - Operating system: 
       Operating System: Arch Linux
       KDE Plasma Version: 5.20.2
       KDE Frameworks Version: 5.75.0
       Qt Version: 5.15.1
       Kernel Version: 5.9.6-arch1-1
       OS Type: 64-bit
       
 - Strawberry Version:
Version 0.8.3

**Comment by jonaski** [comment]
The '&' character is actually stripped off the OSD messages as it caused issues with the native notifications so wherever the problem is, it's likely your tags, ie it has to be "amp;" before it reaches the notifications.

**Comment by jonaski** [comment]
Are you sure you are using version 0.8.3?

**Comment by Kurt-Git** [comment]
When I do a Help -> About Strawberry that is what I get "Version 0.8.3"

As to do about the tags and having "amp;" before It reaches notifications, I'm not sure how. In Strawberry's UI under Artist it displays as "Stevie Ray Vaughan & Double Trouble". In my tag editor, puddletag, it shows as that. Also I delete all ID3 tags as well. My music is all flac and so the tags are flac ones.

**Comment by jonaski** [comment]
I'm looking a bit into this, and I went back and forth a bit on a few changes to the code, but none of them can explain what you are seeing.
I tested version 0.8.3 on KDE and I can't reproduce the behavior with ampersand (&) in the artist tag. A while back I stripped the ampersand character completely from both the summary and the message, and 0.8.3 has that change, but in previous versions it was never encoded as "amp;" anyway, it was incorrectly sent as '&', so even if you had an older version I couldn't explain that behavior. When testing 0.8.3 on KDE here, the ampersand is stripped from both the summary and the message text in the notifications like intended.
I assume you use the native notifications with the artist in the summary text, that's the default setting.
The ampersands are not allowed in the "summary" of the notifications, but it is allowed encoded in the message itself as (amp;).
I changed the behavior now to allow ampersand encoded as "amp;" in the message, this will not affect the problem you are seeing in any way since the artist tag is in the summary by default.
